framework/framework.cpp

```diff
@@ -274,6 +281,8 @@ Framework::Framework(const UString programName, const std::vector<UString> cmdli
 
 	this->threadPool.reset(new ThreadPool(threadPoolSize));
 
+	LogWarning(
+	    ("Current working directory: " + boost::filesystem::current_path().string()).c_str());
```

Comment:
This doesn't look right - a char[] + const char* won't do what you want - you probably want ("Current working directory: %s", boost::filesyste...whatever.c_str());